layouts/partials/header.html

```diff
@@ -42,13 +42,34 @@
   <div class="hidden responsive-container mobile-menu">
     <ul class="">
       {{ range $list }}
-        <li>
+        <li class="flex flex-wrap">
           <a
             href="{{ .URL }}"
-            class="block text-sm px-6 py-2 md:py-3 text-white font-semibold {{ if eq $current.URL .URL }}text-link-orange{{ end }}" title="{{ .Title }}"
+            class="w-9/12 block text-xl px-6 py-2 md:py-3 text-white font-semibold {{ if eq $current.URL .URL }}text-link-orange{{ end }}" title="{{ .Title }}"
           >
             {{ .Name }}
           </a>
+          {{ if .HasChildren }}
+            <div class="chevron-down w-3/12 flex justify-end items-center text-right px-6 py-2 lg:hidden">
+              <img src="/theme-assets/chevron-down.svg" alt="expand list" />
+            </div>
```

Comment:
I think if we embedded the `ul` in this `div`, added something like the styling I suggested above (forgoing the `img` and relying on css), moved the `hidden` to some `@apply` statement (and had a class that toggled the showing of such list), this would be a tad better. But let me know your thoughts?

Reply:
I was able to simplify this by using a div with background instead of the <img>. It worked out fine :)